spec/signet/oauth_2/client_spec.rb

```diff
@@ -63,6 +63,11 @@ def build_form_encoded_response payload
     expect(@client.scope).to eq ["legit", "alsolegit"]
   end
 
+  it "should allow to set granted scopes" do
+    @client.granted_scopes = "granted_scopes1 granted_scopes2"
+    expect(@client.granted_scopes).to eq ["granted_scopes1", "granted_scopes2"]
+  end
```

Comment:
you're missing a test case for passing in an array

Reply:
We're not supporting an array as of now.